modules/services/osm.js

```diff
@@ -10,21 +10,25 @@ import { geoExtent, geoRawMercator, geoVecAdd, geoZoomToScale } from '../geo';
 import { osmEntity, osmNode, osmNote, osmRelation, osmWay } from '../osm';
 import { utilArrayChunk, utilArrayGroupBy, utilArrayUniq, utilObjectOmit, utilRebind, utilTiler, utilQsString } from '../util';
 
+import { osmApiConnections } from '../../config/id.js';
+
 
 var tiler = utilTiler();
 var dispatch = d3_dispatch('apiStatusChange', 'authLoading', 'authDone', 'change', 'loading', 'loaded', 'loadedNotes');
 
+
 var urlroot = 'https://www.openhistoricalmap.org';
```

Comment:
Upstream, c8a3cf154b43469d643b034f47d4e48d449582a1 changed this variable to refer to `osmApiConnections`, but it looks like the merge changed it back. We should customize this URL in id.js rather than here to allow for staging versus production switching.

Reply:
Clicking the “live” button in the status bar switches to the OSM dev server. Clicking the “dev” button switches to the OSM production server. There’s no way to get back to OHM from that point.